packages/ui/src/components/NumberInput/index.tsx

```diff
@@ -191,8 +190,7 @@ export const NumberInput = ({
   'aria-describedby': ariaDescribedBy,
   'data-testid': dataTestId,
 }: NumberInputProps) => {
-  const inputRef =
-    useRef<HTMLInputElement>() as MutableRefObject<HTMLInputElement>
+  const inputRef = useRef<HTMLInputElement>(null)
```

Comment:
```suggestion
  const inputRef = useRef<HTMLInputElement | null>(null)
```

Pourquoi pas celui la ?

Reply:
implicite avec sa valeur par default